server.go

```diff
@@ -2397,6 +2471,11 @@ func (s *server) Start() error {
 			s.connMgr.Stop()
 			return nil
 		})
+
+		// RESOLVE: s.connMgr.Start() is called here, but
```

Comment:
What's up with this `RESOLVE` prefix in the comments?

Reply:
Before this line, it is possible that we get inbound connections that queue up. It's just something that should be documented and probably a test case should be written for it. It is pre-existing and something I noticed when touching this code.